projects/1.20/assets/scorched-guns/scguns/lang/zh_cn.json

```
@@ -0,0 +1,949 @@
+  "scguns.advancement.craft_lightning_rod_connector": "合成延长避雷针",
+  "scguns.advancement.craft_lightning_rod_connector.desc": "用于将雷击从避雷针传导至电池",
+  "scguns.advancement.geothermal_vent": "地热喷口",
+  "scguns.advancement.geothermal_vent.desc": "从海洋深处获取地热喷口",
+  "scguns.advancement.craft_vent_collector": "喷口收集器",
+  "scguns.advancement.craft_vent_collector.desc": "用于从地热喷口收集硝石，或是从硫酸喷口收集硫",
+  "scguns.advancement.craft_polar_generator": "合成极性发电机",
+  "scguns.advancement.craft_polar_generator.desc": "合成极性发电机，用于发电",
+  "scguns.advancement.copper_blueprint": "铜蓝图",
+  "scguns.advancement.copper_blueprint.desc": "用于合成铜枪支",
+  "scguns.advancement.iron_blueprint": "铁蓝图",
+  "scguns.advancement.iron_blueprint.desc": "用于合成铁枪支",
+  "scguns.advancement.treated_brass_blueprint": "涂层黄铜蓝图",
+  "scguns.advancement.treated_brass_blueprint.desc": "揭开奥斯古德博士的秘密，用于合成涂层黄铜枪支",
+  "scguns.advancement.diamond_steel_blueprint": "钻钢蓝图",
+  "scguns.advancement.diamond_steel_blueprint.desc": "发现阿斯加里安的思路，用于合成钻钢枪支",
+  "scguns.advancement.piglin_blueprint": "猪灵蓝图",
+  "scguns.advancement.piglin_blueprint.desc": "从猪灵堡垒中寻得，用于合成猪灵枪支",
+  "scguns.advancement.ocean_blueprint": "海洋蓝图",
+  "scguns.advancement.ocean_blueprint.desc": "从埋藏宝藏中寻得，用于合成海洋枪支",
+  "scguns.advancement.deep_dark_blueprint": "深暗蓝图",
+  "scguns.advancement.deep_dark_blueprint.desc": "从远古城市的深处寻得，用于合成深暗枪支",
+  "scguns.advancement.end_blueprint": "末地蓝图",
+  "scguns.advancement.end_blueprint.desc": "从末地城中寻得，用于合成末地枪支",
+  "scguns.advancement.scorched_blueprint": "熔渣蓝图",
+  "scguns.advancement.scorched_blueprint.desc": "将所有蓝图合并获得，用于合成熔渣枪支",
+  "scguns.advancement.anthralite": "炭金锭",
+  "scguns.advancement.anthralite.desc": "获得炭金，用于制作大多数枪支",
+  "scguns.advancement.niter_dust": "硝石粉",
+  "scguns.advancement.niter_dust.desc": "获得硝石，用于生产火药，也能用于降温",
+  "scguns.advancement.ancient_brass": "远古黄铜",
+  "scguns.advancement.ancient_brass.desc": "获得远古黄铜，用于制作黄铜弹壳和涂层黄铜",
+  "scguns.advancement.sulfur_chunk": "硫碎块",
+  "scguns.advancement.sulfur_chunk.desc": "获得硫，用于生产火药",
+  "scguns.advancement.treated_brass": "涂层黄铜",
+  "scguns.advancement.treated_brass.desc": "获得涂层黄铜，用于制作涂层黄铜枪支",
+  "scguns.advancement.diamond_steel": "钻钢",
+  "scguns.advancement.diamond_steel.desc": "获得钻钢，用于制作钻钢枪支",
+  "scguns.advancement.scorched_ingot": "熔渣锭",
+  "scguns.advancement.scorched_ingot.desc": "获得熔渣锭，用于制作熔渣枪支",
+  "scguns.advancement.gunpowder": "火药",
+  "scguns.advancement.gunpowder.desc": "获得火药！弹药制作中最基础的混合物",
+  "scguns.advancement.peal": "鸣雷火药",
+  "scguns.advancement.peal.desc": "获得鸣雷火药！一种含有幽匿成分的火药混合物，用于制作进阶弹药",
+  "scguns.advancement.sheol": "冥世火药",
+  "scguns.advancement.sheol.desc": "获得冥世火药！一种下界的火药混合物，用于制作进阶弹药",
+  "scguns.advancement.nitro_powder": "硝化火药",
+  "scguns.advancement.nitro_powder.desc": "获得硝化火药！最高级的火药，只用于最为强大的弹药",
+  "scguns.advancement.vehement_coal": "炽烈煤",
+  "scguns.advancement.vehement_coal.desc": "获得炽烈煤，一种烈性燃料",
+  "scguns.advancement.thermolith": "加热机",
+  "scguns.advancement.thermolith.desc": "合成加热机,使用炽烈煤供能，可以将岩石融化成熔岩",
```

```suggestion
  "scguns.advancement.thermolith.desc": "合成加热机，使用炽烈煤供能，可以将岩石融化成熔岩",
```
【要求】
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。